integration/tests/basic/rpc/test_rpc_estimate_gas.py

```diff
@@ -0,0 +1,173 @@
+import typing as tp
+
+import allure
+import pytest
+
+from integration.tests.basic.helpers import rpc_checks
+from integration.tests.basic.helpers.basic import BaseMixin
+from utils import helpers
+
+
+@pytest.fixture(scope="class")
+def common_contract(web3_client, class_account) -> tp.Any:
+    common_contract, _ = web3_client.deploy_and_get_contract(
+        "common/Common", "0.8.12",
+        contract_name="Common", account=class_account
+    )
+    yield common_contract
+
+
+@allure.feature("JSON-RPC-ESTIMATE-GAS validation")
```

Comment:
I checked and found that all feature in basic/rpc is now different, please fix too.
feature should be one for full category: JSON-RPC validation
And story should be differents

Reply:
Done